### ISSUE TITLE
Mixed model tool feedback

### ISSUE THREAD

**Issue by ch728** [solgenomics/sgn#3248]
I'm wondering if it would be possible to allow the design of models to be reusable. For example, the breeding program I work with only needs to run a handful of different mixed model analyses. I would like to design the models ahead of time and then the users could select one of my predefined models to run on their data set. That way the user only needs to know which model is appropriate for their data set  and what their dependent value should be, and not the details of what independent variables should be included as random/fixed/interaction effects. The specification of the model and the application of the model in production are two separate concerns in this scenario.

**Comment by lukasmueller** [comment]
Yes, this was the intention from the beginning, to be able to store the model and retrieve it in the tool itself or in some other scenario. Currently, you can save the model, but the tool cannot retrieve them again. This is not totally trivial to implement but I think it is a necessary feature :-)
